tests/integration/test_register_persister.py

```diff
@@ -53,3 +68,22 @@ def test_load_cassette_with_custom_persister(tmpdir, httpbin):
     with my_vcr.use_cassette(test_fixture, serializer="json"):
         response = urlopen(httpbin.url).read()
         assert b"difficult sometimes" in response
+
+
+def test_load_cassette_with_custom_persister(tmpdir, httpbin):
```

Comment:
How about a rename to `test_load_cassette_persister_exception_handling`? It think something like that would be a bit more clear and in line with the docstring.

Reply:
Done! I just realized I forgot to rename the function after copy-pasting, so this fixes the name collision as well.